src/site/styles/layout.less

```diff
@@ -93,11 +93,46 @@ input {
     position: relative;
 }
 
-.action-confirmation {
-    position: absolute;
-    top: 0;
-    left: 0;
-    background: white;
+.action-confirmation { 
```

Comment:
Migrating comment:
> Why nest the rules under `.action-confirmation`? You don't need the extra specificity, it just adds CSS bloat.

Migrating response:
> More specificity is better, it adds the distinction that these rules are not to be used on other components and are specific to a single component.

Actually, more specificity can become a performance concern. The following LESS:

```less
a {
    b { ... }
    c { ... }
}

becomes the following CSS:

```css
a b { ... }
a c { ... }
```

Something to keep in mind. Consider this thread resolved as there are only two rules here.